docker/Dockerfile

```diff
@@ -0,0 +1,24 @@
+#
+# Copyright (c) 2019
+# Intel
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+FROM golang:1.11.9-alpine
```

Comment:
Our other dockerfiles use `FROM golang:1.11-alpine` to align with the latest Go 1.11 build when it arrives. Do we want to do the same thing here?

Reply:
Currently we are building in go version 1.11.5 for the modules. I chose this to match the current golang base image that we have for golang in [ci-build-images](https://github.com/edgexfoundry/ci-build-images/blob/master/golang/Dockerfile). The idea behind these build images is to have a set of standard build images so people can use them in their FROM statements and not have repos building in different versions of golang.

Eventually we will have a staged golang base image in the EdgeX nexus repo which this dockerfile will use in it's FROM statement. There probably is some required discussion around which version to use in the base image because there are a lot discrepancies across the repos.